### PR TITLE
Follow API changes

### PR DESCRIPTION
**Pull request by csillag**
This builds on top of #267 , and fixes that's broken.

A notable new feature is that TX status can now also be unknown. @donouwens has created new designs for that, and this PR also incorporates that.

### CODE REVIEW

**Comment by csillag** [comment]
Here is two screenshots of the new "unknown" state:

In the details view:

image

On transaction list:

image


Design feedback from @donouwens:

>  All in all looks good to me!

File: src/app/components/TransactionStatusIcon/index.tsx [hunk 1]
```diff
@@ -1,48 +1,80 @@
...
+  unknown: COLORS.grayMedium,
+  success: COLORS.eucalyptus,
+  failure: COLORS.errorIndicatorBackground,
+}
+
+const statusIcon: Record<TxStatus, () => ReactNode> = {
+  unknown: () => <Help color={'inherit'} fontSize="inherit" />,
+  success: () => <CheckCircleIcon color="success" fontSize="inherit" />,
+  failure: () => <CancelIcon color="error" fontSize="inherit" />,
+}
```
Comment (lukaw3d) [inline src/app/components/TransactionStatusIcon/index.tsx:28]: Is there a benefit to generating these every time?

**Comment by csillag** [comment]
This is so old, I will just close and re-create it.

**Comment by csillag** [comment]
All comments from here has been addressed in the follow-up PR, #340.

Except this one:

> RuntimeTransactionMethod is no longer a good name

Does anyone have a better suggestion?

**Pull request closed without merging**